src/apphosting/githubConnections.ts

```diff
@@ -180,6 +195,107 @@ async function createFullyInstalledConnection(
   return conn;
 }
 
+async function manageInstallation(connection: devConnect.Connection): Promise<void> {
+  utils.logBullet(
+    "Manage the Firebase App Hosting GitHub app to enable access to GitHub repositories",
+  );
+  const targetUri = connection.githubConfig?.installationUri;
+  if (!targetUri) {
+    throw new Error("No installation given to manage");
```

Comment:
should we be throwing `FirebaseError` here?

If this is a user facing error, is there any guidance you can provide to help user "debug" this state (probalby just.. to try again?)

Reply:
ya i think `FirebaseError` makes sense here and the error message here could be better. Something like "Failed to get installation URI. Please try again."?